FWCore/Utilities/interface/InputTag.h

```diff
@@ -39,6 +39,8 @@ namespace edm {
     ///product with the label and instance
     std::string const& process() const {return process_;} 
 
+    bool isUnitialized() const {return label_.empty() && instance_.empty() && process_.empty();}
```

Comment:
In practice, you only have to test `label_.empty()` since a valid InputTag must always have at least that one set.

Reply:
thats a good point, should have realised it, will make the change in the next update. This makes a little less of an important change as I could just do label().empty(). I'm totally neutral on this change (hence why its a seperate commit), I thought it was nice to have a little syntax sugar to telegraph the intention but I dont mind if others feel label().empty() is the way to go and avoid polluting the number of methods.